### ISSUE TITLE
Access function integration

### ISSUE THREAD

**Issue by CouleeApps** [Submitty/Submitty#2384]
After #2337 I need to convert the other controllers to use the Access class.

**Comment by bmcutler** [comment]
status?

**Comment by CouleeApps** [comment]
Low priority at the moment. The rest of the codebase is basic "getGroup() == Instructor" checks that would be pretty easy to add to Access but would just take a while.

**Comment by bmcutler** [comment]
closing, can be re-opened with more specific instructions & files that need refactor, etc.
